e2e/test/Helpers/templates/FaultInjection.cs

```diff
@@ -39,17 +39,17 @@ public static class FaultInjection
         public const string FaultCloseReason_Boom = "boom";
         public const string FaultCloseReason_Bye = "byebye";
 
-        public const int DefaultDelayInSec = 5; // Time in seconds after service initiates the fault.
-        public const int DefaultDurationInSec = 5; // Duration in seconds
-        public const int LatencyTimeBufferInSec = 10; // Buffer time waiting fault occurs or connection recover
+        public static readonly TimeSpan DefaultDelayInSec = TimeSpan.FromSeconds(5); // Time in seconds after service initiates the fault.
+        public static readonly TimeSpan DefaultDurationInSec = TimeSpan.FromSeconds(5); // Duration in seconds
+        public static readonly TimeSpan LatencyTimeBufferInSec = TimeSpan.FromSeconds(10); // Buffer time waiting fault occurs or connection recover
 
-        public const int WaitForDisconnectMilliseconds = 3 * DefaultDelayInSec * 1000;
-        public const int WaitForReconnectMilliseconds = 2 * DefaultDurationInSec * 1000;
-        public const int ShortRetryInMilliSec = (int)(DefaultDurationInSec / 2.0 * 1000);
+        public static readonly TimeSpan WaitForDisconnectSeconds = TimeSpan.FromSeconds(15); // 3 * DefaultDelayInSec
```

Comment:
You could do the TimeSpan math if you really wanted to, right?

Reply:
I actually didn't even think of that!